pkg/pillar/go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/jaypipes/ghw v0.8.0
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.5.0
 	github.com/lf-edge/edge-containers v0.0.0-20240207093504-5dfda0619b80
-	github.com/lf-edge/eve-api/go v0.0.0-20240424223403-4feef58259f3
+	github.com/lf-edge/eve-api/go v0.0.0-20240520195755-0462d38358c9
```

Comment:
There's a change to introduce flag allow_to_discover to Network Instance I'm afraid :c 

If you can create separate PR bumping EVE API version I can can rebase on master, so you'll get your satisfaction unlike what Rolling Stones sang :D

Reply:
Eve already uses eve-api commit https://github.com/lf-edge/eve-api/commit/4feef58259f30f1a57d175af8ed710edd9e3d5d1
I do not see anything other than MTU changes since then: https://github.com/lf-edge/eve-api/commits/main/

Reply:
Oh, then I shall remove this immediately!

Reply:
But that means, someone bumped EVE API with my changes :c Oh well

Reply:
Because you didn't complain :)